<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="LAPTOP-U0UME3N7" failures="2" tests="2" name="Default test(failed)(failed)" time="0.104" errors="0" timestamp="2022-09-26T15:00:15 IST">
  <testcase classname="Assertion" name="assertEquals" time="0.025">
    <failure type="java.lang.AssertionError" message="expected [assertion1] but found [assertion]">
      <![CDATA[java.lang.AssertionError: expected [assertion1] but found [assertion]
at Assertion.assertEquals(Assertion.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- assertEquals -->
  <testcase classname="Assertion" name="assertNotEquals" time="0.002">
    <failure type="java.lang.AssertionError" message="did not expect [This assertion will pass] but found [This assertion will pass]">
      <![CDATA[java.lang.AssertionError: did not expect [This assertion will pass] but found [This assertion will pass]
at Assertion.assertNotEquals(Assertion.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- assertNotEquals -->
  <testcase classname="Assertion" name="assertNull" time="0.0">
    <ignored/>
  </testcase> <!-- assertNull -->
  <testcase classname="Assertion" name="assertNotNull" time="0.0">
    <ignored/>
  </testcase> <!-- assertNotNull -->
  <testcase classname="Assertion" name="assertTrue" time="0.0">
    <ignored/>
  </testcase> <!-- assertTrue -->
  <testcase classname="Assertion" name="assertFalse" time="0.0">
    <ignored/>
  </testcase> <!-- assertFalse -->
</testsuite> <!-- Default test(failed)(failed) -->
